src/admin/components/UserResumeTable/index.tsx

```diff
@@ -13,16 +13,40 @@ interface UserResumeTableProps {
 
 const UserResumeTable: React.FC<UserResumeTableProps> = (props) => {
   const { resumes } = props;
+  const [yearFilters, setYearFilters] = useState([{ text: '2026', value: '2026' }]);
   const [selectedRowKeys, setSelectedRowKeys] = useState<React.Key[]>([]);
 
+  const updateYearFilters = () => {
+    setYearFilters(
+      resumes
+        .filter((value: UserResume, index: number) => {
+          // Resumes.indexOf(value) === index checks for first occurance
+          return resumes.indexOf(value) === index;
```

Comment:
Single-line arrow functions can be inlined to save space and readability.

Example

From
```JavaScript
mylist.map((element) => {
   return element * 2
})
```

To
```JavaScript
mylist.map(element => element * 2)
```
Note: The thing on the right side of the arrow is the return value. If you need any additional lines of code in the body of the function, this won't work

Reply:
ESLint edited it to be like so. Let me know if this is OK.
```
resumes
        .filter((value: UserResume, index: number) => resumes.indexOf(value) === index)
        .sort()
        .map((element) => ({ text: `${element.user.graduationYear}`, value: `${element.user.graduationYear}` })),
    );
```